alumni/index.html

```diff
@@ -1,7 +1,27 @@
 ---
 layout: page
-title: Coaches
+title: Pictures
 ---
+<div id="batch1" class="section">
+    <div class="container">
+        <div class= "row">
+            <img src="/img/batch-3-pic6.jpg" class="col s7 m7">
+            <div class="col s12 m5">
+                <h4 >Batch-3</h4>
+                <p>The third batch of Hackership ran in Berlin, at the Wooga offices, and took place from July till September 2015.<a href="https://github.com/Hackership/alumni/blob/master/batch-1.md"> These great people came to learn with us.</a></p>
```

Comment:
This currently links to the wrong file. Even if we don't have the new one yet and it will give us a 404 for now, let's please link to batch-3 because I know we'll forget to fix this later...